### ISSUE TITLE
iOS Notifications Regression

### ISSUE THREAD

**Issue by brunobowden** [WorldHealthOrganization/app#1855]
## Summary

This was previously verified with `v0.15.0` with sending FCM Test Messages. Notifications aren't working in the `1.0.0` release and not even `v0.15.0` any more. At the same time, I'm not aware of any settings changes in Firebase that might cause this. The Android notifications work.

## System

- iPhone 11 Pro Max with iOS 14.2

## Checklist:

- [x] Searched the [existing issues](https://github.com/WorldHealthOrganization/app/issues) to ensure you are not creating a duplicate.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).

**Comment by advayDev1** [comment]
Q: will not fixing this in this version means ios persons opted in to notifications would never receive notifications if they dont launch their app after a later update?  

Is it certain that the build flavor code is working and binding the app to the prod api url and prod firebase project id?

**Comment by brunobowden** [comment]
I'm sure it's getting the right flavor as for any flavor other than prod, it will display the very visible "No privacy - {staging|hack} server" alert.

For the moment we aren't sending any notifications, so I expect us to send out and push updates for a new build before doing so. I don't yet know all the implications.

cc @travelkev

**Comment by advayDev1** [comment]
That alert is based on the flavor constant IIRC.  I'm asking if the google-services config file in the final binary is the prod. @brunobowden

**Comment by brunobowden** [comment]
Yes, it has to be. If it wasn't the `prod` config then it would show the "No privacy" alert, as it does for the `staging` and `hack` servers. I've checked that there's no alert for rc1 on either Android or iOS.

**Comment by matthewblain** [comment]
Can you verify that the putClientSettings is being called correctly from iOS? Should be platform-indepdendent, but I did not have a way to test it there.

**Comment by brunobowden** [comment]
`putClientSettings` is being called on iOS.... additional confirmation that it's got the correct `google-services.json` file. Copying the token from that RPC and sending a test message in the Firebase console fails. Repeating that same procedure for Android works. Leading me to suspect it's an auth or build issue on iOS.
